### PR TITLE
[VPP] TGL VPP features merge

### PR DESCRIPTION
**Pull request by fzhar**

### CODE REVIEW

File: _studio/mfx_lib/vpp/include/mfx_vpp_main.h [hunk 1]
```diff
@@ -116,6 +116,8 @@ class VideoVPPMain : public VideoVPP {
                 nativeSurface->Data.FrameOrder = surface->Data.FrameOrder;
                 nativeSurface->Data.TimeStamp = surface->Data.TimeStamp;
                 nativeSurface->Info = surface->Info;
+                nativeSurface->Data.ExtParam = surface->Data.ExtParam;
+                nativeSurface->Data.NumExtParam = surface->Data.NumExtParam;
```
Comment (dmitryermilov) [inline _studio/mfx_lib/vpp/include/mfx_vpp_main.h]: THIS IS A BUG! Please do git blame

Comment (fzhar) [reply _studio/mfx_lib/vpp/include/mfx_vpp_main.h]: Fixed (your latest commit is merged here)

File: _studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp [hunk 2]
```diff
@@ -4695,6 +4749,52 @@ mfxStatus ValidateParams(mfxVideoParam *par, mfxVppCaps *caps, VideoCORE *core,
 
         switch(id)
         {
+        case MFX_EXTBUFF_VPP_MIRRORING:
+        {
+            mfxExtVPPMirroring* extMir = (mfxExtVPPMirroring*)data;
+            // SW mirroring supports only horizontal mode
+            mfxU32 maxMirrorSupportMode = 1;
```
Comment (dmitryermilov) [inline _studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp]: what is '1' and '2'? Please use named constants

**Comment by dvrogozh** [comment]
update PR to pass CI. I guess you need to change copyright year...

**Comment by fzhar** [comment]
checks are passed

File: _studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp (hunk 2 above)
Comment (dvrogozh) [reply _studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp]: not addressed.

Comment (fzhar) [reply _studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp]: fixed

**Comment by dmitryermilov** [comment]
Please provide a list of new/fixed feature. It's essential for RN.

**Comment by fzhar** [comment]
> Please provide a list of new/fixed feature in the commit message. It's essential for RN.

Done

**Pull request merged**